plugins/legacy-preset-chart-nvd3/src/Pie/controlPanel.ts

```diff
@@ -45,6 +45,7 @@ export default {
                 ['percent', 'Percentage'],
                 ['key_value', 'Category and Value'],
                 ['key_percent', 'Category and Percentage'],
+                ['key_value_percent', 'Category Value and Percentage'],
```

Comment:
NIT: + `,` as in `'Category, Value and Percentage'`

Reply:
done